js/index.js

```diff
@@ -1 +1,66 @@
 // Your code goes here
+// `mouseover`
+const nav = document.querySelector('.nav')
+
+nav.addEventListener('mouseover', function(event) {
+  event.target.style.color = 'orange';
+  event.target.style.fontWeight = 900;
+  setTimeout(function() {
+    event.target.style.color = "";
+    event.target.style.fontWeight = '';
+  }, 500);
+}, false);
+
+// `keydown`
+
+const home = document.querySelector('.home')
+
+mainNav.addEventListener('keydown', function(event) {
+  event.style.backgroundColor = #B7D9E3;
+  setTimeout(function() {
+    event.style.backgroundColor = '';
+  }, 200);
+}, false);
+
+// `wheel`
+
+const contentTop = document.querySelector('content-selection > .text-content')
+
+contentTop.addEventListener('wheel', function(event) {
+    event.target.style.fontSize = "40px";
+    setTimeout(function() {
+      event.target.style.fontSize = "";
+    }, 500);
+}, false);
+
+// `drag / drop`
```

Comment:
I don't mind seeing your attempts at this, I feel like I'm only seeing the finished version. I'm here to give feedback, so even if it's not working, it'd be nice to see so I can give you feedback.